pkg/cache/secretcache.go

```diff
@@ -0,0 +1,150 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cache
+
+import (
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/dapr/components-contrib/secretstores"
+
+	"github.com/dgraph-io/ristretto"
+	"github.com/pkg/errors"
+)
+
+const (
+	separator = "||"
```

Comment:
Is there a doc to comment this?

Reply:
This is an implementation detail and is used only inner this package. Need a separator to support multiple versions of a key, and this is widely used as a separator in dapr.

Reply:
OK.